aospy/test/test_data_loader.py

```diff
@@ -492,6 +492,24 @@ def preprocess(ds, **kwargs):
         expected = xr.open_dataset(filepath)['condensation_rain']
         np.testing.assert_allclose(result.values, expected.values)
 
+    def test_load_variable_mask_and_scale(self):
+        def preprocess(ds, **kwargs):
+            # This function converts all values to missing values
+            ds['condensation_rain'] = 0. * ds['condensation_rain'] + 1.0e20
+            ds['condensation_rain'].attrs['_FillValue'] = 1.0e20
+            return ds
+
+        self.data_loader.preprocess_func = preprocess
+
+        result = self.data_loader.load_variable(
+            condensation_rain, datetime(5, 1, 1),
+            datetime(5, 12, 31),
+            intvl_in='monthly')
+
+        result = np.isfinite(result).sum().item()
```

Comment:
Either rename this or the above `result` to a different name.  E.g. for this one `result_integral`.